cookiecutter/exceptions.py

```diff
@@ -1,104 +1,84 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-"""
-cookiecutter.exceptions
------------------------
-
-All exceptions used in the Cookiecutter code base are defined here.
-"""
+"""All exceptions used in the Cookiecutter code base are defined here."""
 
 
 class CookiecutterException(Exception):
-    """
-    Base exception class. All Cookiecutter-specific exceptions should subclass
-    this class.
+    """Base exception class.
+
+    All Cookiecutter-specific exceptions should subclass this class.
     """
 
 
 class NonTemplatedInputDirException(CookiecutterException):
-    """
-    Raised when a project's input dir is not templated.
```

Comment:
I have a preference for the summary line of [multi-line docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) to be on the next line.
>  The summary line may be on the same line as the opening quotes _or on the next line_. 

Thoughts @audreyr @pydanny @hackebrot